composer.json

```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "sonata-project/media-bundle": "~2.3|~3.0"
+        "sonata-project/media-bundle": "~3.0"
```

Comment:
You have to add a hard dependency to `"sonata-project/admin-bundle": "~3.1"` because of the `Sonata\AdminBundle\Admin\AbstractAdminExtension` class.